plugin-build/src/test/kotlin/io/sentry/android/gradle/instrumentation/VisitorTest.kt

```diff
@@ -82,11 +82,14 @@ class VisitorTest(
 
     @After
     fun printLogs() {
-        tmpDir.root.listFiles()
-            ?.filter { it.name.contains("instrumentation") }
-            ?.forEach {
-                print(it.readText())
-            }
+        // only print bytecode when running locally
+        if (System.getenv("CI")?.toBoolean() != true) {
```

Comment:
If it's just a test, why is that filtered? if it fails on CI, we cannot see the output

Reply:
yeah but if it doesn't fail, it becomes too noisy, so you're gonna have a hard time looking for the real problem if something else fails. And if this fails, you anyway would checkout and run locally to fix the problem, I don't think one could fix bytecode issues by just looking at the CI output :D 

As an alternative, we could use upload-action to upload output of the bytecode visitors as a zip archive

Reply:
filed https://github.com/getsentry/sentry-android-gradle-plugin/issues/259